Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -55,6 +58,9 @@ namespace Hazel {
 
 		glfwSetWindowUserPointer(m_Window, &m_Data);
 		SetVSync(true);
```

Comment:
Not about this PR, but this small fix we can do in here:
this should be `SetVSync(props.VSync)`